license_header.txt

```diff
@@ -1,5 +1,5 @@
 /*!
 Plottable @VERSION (https://github.com/palantir/plottable)
-Copyright 2014-2017 Palantir Technologies
+Copyright 2014-present Palantir Technologies
```

Comment:
Facebook uses this style in their react headers: e.g. https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react.js